pykit/transport/db2/Db2Connection.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import json
+from ...Toolkit import Toolkit
+import ibm_db_dbi as dbi
+
+
+
+
+class Db2Connection:
+    """
+    Represents a REST HTTP connection that can be used to get the Python
+    Toolkit object.
+    """
+
+    def __init__(self, database, username, password):
+        
+        self.connection = dbi.connect(database=database, \
+                       user=username, password=password)
```

Comment:
It's possible to just wrap `ibm_db_dbi.connect`:

```python
def __init__(self, *args, **kwargs):
    self.connection = dbi.connect(args, kwargs)
```

https://www.digitalocean.com/community/tutorials/how-to-use-args-and-kwargs-in-python-3

This would allow users to pass any connection options they want, though is that something we actually want them to be able to do?

Reply:
Well, probably. Thoughts on having db, username, and password explicit (for comprehension), and adding **kwargs on the end?

Reply:
Yeah, you can have as many named parameters as you like followed by `*args` and `**kwargs`, however this would mess up positional argument handling unless you had the same named arguments and in the same order as `ibm_db_dbi.connect`. So I'd suggest only supporting `kwargs`.

Does that make sense?